boot/boot.js

```diff
@@ -1444,6 +1444,7 @@ $tw.loadTiddlersBrowser = function() {
 	for(var t=0; t<containerIds.length; t++) {
 		$tw.wiki.addTiddlers($tw.wiki.deserializeTiddlers("(DOM)",document.getElementById(containerIds[t])));
 	}
+	callback();
```

Comment:
The non-callback API cannot be converted to async by external code, whereas this allows it to be async'd before calling `$tw.boot.boot()`. 

It seems to me that that only applies as a matter of principle. It does not matter here that we are sync'ly calling the callback, because the callback code is in boot.js and therefore is under our control anyway. It was originally sync, and it still is that way.